packages/core/package.json

```diff
@@ -1,13 +1,20 @@
 {
-  "name": "@loopback/loopback",
+  "name": "@loopback/core",
   "version": "4.0.0-alpha.1",
   "description": "",
   "main": "index",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "tsc --target es2017 --outDir lib",
+    "build:lib6": "tsc --target es2015 --outDir lib6",
+    "clean": "rm *.tgz; rm -rf ./package",
+    "prepublish": "npm run build",
+    "pretest": "npm run build",
     "integration": "mocha --opts ../../test/mocha.opts 'test/integration/**/*.ts'",
     "test": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts' 'test/integration/**/*.ts' 'test/acceptance/**/*.ts'",
-    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'"
+    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'",
+    "verify": "npm run clean && npm pack && tar xvzf *.tgz && tree package"
```

Comment:
IMO, it's much easier to make a mistake without tree since we're verifying by eye (only internal users are publishing anyways, so it seems safe to assume we're the only ones using this command?)